client/src/components/profile/components/HeatMap.js

```diff
@@ -57,18 +68,30 @@ function HeatMap({ calendar, streak }) {
       <FullWidthRow>
         <CalendarHeatMap
           classForValue={value => {
-            if (!value) {
+            if (value.count < 1) {
```

Comment:
```suggestion
            if (!value || value.count < 1) {
```
Just in case value somehow doesn't get set.